tests/scripts/all.sh

```diff
@@ -400,6 +400,18 @@ fi
 
 fi # MemSan
 
+msg "build: cmake build from subdirectory"
```

Comment:
It's called an ['out-of-source' build directory](https://cmake.org/Wiki/CMake_FAQ#Out-of-source_build_trees).

Can you change the name to `cmake 'out-of source' build`